### DISCUSSION TITLE
[Question][Hw3] Unbound value __dbz_sanitizer__

### DISCUSSION THREAD

**Discussion by 0xXcrypted** [prosyslab-classroom/cs348-information-security#545] [Q&A]
I'm stuck using Llvm.declare_function
`let f1 = Llvm.declare_function __dbz_sanitizer__ [|Llvm.i32_type llctx; Llvm.i32_type llctx; Llvm.i32_type llctx|] llm in`
and this gives me `Error: Unbound value __dbz_sanitizer__`
How can I solve this issue?

**Comment by mickowale** [comment]
The first argument to `Llvm.declare_function` must be a string.

**Comment by 0xXcrypted** [comment]
oh my... thanks for your help
what a shame
